tests/MongoDBDriverCursor_setTypeMap.phpt

```diff
@@ -7,19 +7,15 @@ $m = new MongoDB\Driver\Manager("mongodb://localhost:27017");
 $c = new MongoDB\Driver\Command( [ 'drop' => 'test'] );
 try {
 	$cursor = $m->executeCommand( 'demo', $c );
-
-	foreach( $cursor->toArray() as $result )
-	{
-		var_dump($result);
```

Comment:
Dumping the result should not be necessary, as libmongoc reports an error (i.e. exception is thrown) if the command fails.